src/auth.ts

```diff
@@ -30,5 +30,16 @@ export const {
           ]
         : []),
     ],
+  callbacks: {
+    async session({session}) {
```

Comment:
Can you explain what this is doing?

Reply:
this returns additional details of logged in user when using useSession in client component

Reply:
like userID which is needed for matching the recGames

Reply:
Does this risk exposing any user credentials in the client? I am assuming it is ok.

Reply:
It gives userId, name, email, pic_url, sessionToken.